src/lspsrc/vector.75

```diff
@@ -0,0 +1,324 @@
+;;;  VECTOR    				-*-Mode:Lisp;Package:SI;Lowercase:T-*-
+;;;  **************************************************************************
+;;;  *** MacLISP ******** VECTOR support **************************************
+;;;  **************************************************************************
+;;;  ******** (c) Copyright 1982 Massachusetts Institute of Technology ********
+;;;  **************************************************************************
+
+(herald VECTOR /74)
```

Comment:
75

Reply:
Now here, I actually changed VECTOR 74 to get it to compile.  So here, I'd be comfortable either a) removing the version number from the HERALD or b) updating it to 75.  Recommendations?